Sources/Core/Internal/BundleImageLoader.swift

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Imagine Engine
+ *  Copyright (c) John Sundell 2017
+ *  See LICENSE file for license
+ */
+
+import Foundation
+import CoreGraphics
+
+class BundleImageLoader: TextureImageLoader {
```

Comment:
This class can be `final`. Also, I suggest naming it the full `BundleTextureImageLoader`. Nitpicky detail, but that'll make it nice and clear that this is an image loader for *textures*.

Reply:
Also, another nitpicky detail, but in frameworks I usually add an explicit `internal` to the classes. This to make it crystal clear to others (like yourself 😄) that this class is only meant for internal use.